File: Blackoffice/Models/COMPANYPROFILE_Models.vb

Imports MD
Public Class COMPANYPROFILE_Models
    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of COMPANYPROFILE_Data)
    End Class

    Public Class COMPANYPROFILE_Data
        Public Property COMID As Integer
        Public Property BRNID As String
        Public Property BRNNAMETHAI As String
        Public Property BRNNAMEENG As String
        Public Property BRNTAXID As String
        Public Property BRNVAT As Double
        Public Property COMPNAME As String
        Public Property ADDRESS1 As String
        Public Property ADDRESS2 As String
        Public Property REMARK As String
        Public Property GATEWAY As Boolean
        Public Property LOCATION As String
        Public Property DIVISION As String
        Public Property COMPANYCODE As String
        Public Property BRNID2 As String
        Public Property LOCATION2 As String
        Public Property SERVICECHARGE As Double
        Public Property FullTaxNo As String
        Public Property TCustCode As String
    End Class




End Class

File: TouchPOS_API/Models/COMPANYPROFILE_Models.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class COMPANYPROFILE_Models
    {

        public class COMPANYPROFILE_data
        {

            public int COMID { get; set; }

            public string BRNID { get; set; }

            public string BRNNAMETHAI { get; set; }

            public string BRNNAMEENG { get; set; }

            public string BRNTAXID { get; set; }

            public double BRNVAT { get; set; }

            public string COMPNAME { get; set; }

            public string ADDRESS1 { get; set; }

            public string ADDRESS2 { get; set; }

            public string REMARK { get; set; }

            public bool GATEWAY { get; set; }

            public string LOCATION { get; set; }

            public string DIVISION { get; set; }

            public string COMPANYCODE { get; set; }

            public string BRNID2 { get; set; }

            public string LOCATION2 { get; set; }

            public double SERVICECHARGE { get; set; }

            public string FullTaxNo { get; set; }

            public string TCustCode { get; set; }
            public COMPANYPROFILE_data()
            {
                LOCATION = "";
                DIVISION = "";
                COMPANYCODE = "";
                BRNID2 = "";
                LOCATION2 = "";
                SERVICECHARGE = 0;
                LOCATION = "";
                TCustCode = "";

            }
        }

       
}
}